eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,24 @@
+parameters: 
+  TargetUserVariable: ''
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+variables:
+    - template: ../variables/tool-version.yml
+steps:
+  - pwsh: |
+      dotnet tool install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version '$("Azure.Sdk.Tools.RetrieveCodeOwners")' "Azure.Sdk.Tools.RetrieveCodeOwners"
+    displayName: 'Install tool RetrieveCodeOwners'
+    workingDirectory: '$(Agent.ToolsDirectory)'
+
+  - pwsh: |
+      Push-Location '$(Agent.ToolsDirectory)'
+      .\retrieve-codeowners `
+        --target-directory "/sdk/${{ parameters.ServiceDirectory }}/" `
```

Comment:
Assuming this gets combined with `--root-directory`, I'd still remove the leading slash, as it's clearer that this is a relative path.

Reply:
This is align with CODEOWNER file format: 
https://github.com/Azure/azure-sdk-for-python/blob/main/.github/CODEOWNERS#L13

Reply:
FWIW we end up trimming the slashes in our codeowner matching.